test/unit/uk/gov/hmrc/tariffclassificationfrontend/connector/BindingTariffClassificationConnectorSpec.scala

```diff
@@ -44,7 +46,9 @@ class BindingTariffClassificationConnectorSpec extends UnitSpec with WiremockTes
   private val otherQueue = Queue(2, "other", "Other")
   private implicit val hc = HeaderCarrier()
 
-  given(configuration.bindingTariffClassificationUrl).willReturn("http://localhost:20001")
+  override def beforeEach(): Unit = {
```

Comment:
I'd also argue this should be below the connector construction (below)...

Reply:
removed